Zend/zend_stack.h

```diff
@@ -28,6 +28,11 @@ typedef struct _zend_stack {
 
 #define STACK_BLOCK_SIZE 16
 
+typedef enum {
+	ZEND_STACK_APPLY_TOPDOWN,
```

Comment:
Looks correct in general, just curiosity question : what led you to change the original values ?
It seems safe enough doing so, I do not think it will bite later on just curous..

Reply:
I reviewed all code locations where they are used, also checked the commit which added them (9c754da0fcec0ca1c466618e4605c69d24745350), and figured these were arbitrary values with no meaning. I also figured that zero (for which there is no macro) isn't a special case, for example by casting to `bool` somewhere or relying on zero-initialization (via `memset()` or `bss`).
Then I decided to not specifiy any explicit values at all, because that would indicate to the next guy that the values are important, when they really aren't. I decided for simplicity.

Reply:
sounds fair.

Reply:
> I reviewed all code locations where they are used, also checked the commit which added them ([9c754da](https://github.com/php/php-src/commit/9c754da0fcec0ca1c466618e4605c69d24745350)), and figured these were arbitrary values with no meaning. I also figured that zero (for which there is no macro) isn't a special case, for example by casting to `bool` somewhere or relying on zero-initialization (via `memset()` or `bss`). Then I decided to not specifiy any explicit values at all, because that would indicate to the next guy that the values are important, when they really aren't. I decided for simplicity.

please - add this to commit message. and  pretty please, with sugar on top - write BIG commit messages to explain next guy WHY this is done. You do a lot of good work, but not explain what and why was done.